internal/pkg/deploy/cloudformation/errors.go

```diff
@@ -39,6 +42,15 @@ func (err *ErrTemplateNotFound) Error() string {
 	return fmt.Sprintf("failed to find the cloudformation template at %s", err.templateLocation)
 }
 
+func (e *ErrTemplateNotFound) Is(target error) bool {
```

Comment:
👍

Reply:
Needs a doc string sine this is exported, yes?

Reply:
The `Is` and `As` methods were [introduced in go1.13](https://blog.golang.org/go1.13-errors#TOC_4.) and I believe don't need a docstring.

But added one anyway :)